### PR TITLE
Awesome book class

### PR DESCRIPTION
**Pull request by Kaghenimbale**
- [ ]  Use JavaScript classes.
- [ ]  Maintain all of the original functionality of the application
- [ ]  Add CSS styles to the application to make

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,69 +1,82 @@
...
+      this.id = 1;
+    }
+  }
 
-    const book = `
-    <ul class=''>
-    <li>${obj.Title}</li>
-    <li>${obj.Author}</li>
-    </ul>
+  AddBook() {
```
Comment (bobb-Rob) [inline script.js]: - I  tried to add book but not response when the  add book submit button is clicked. Please fix this.

File: script.js [hunk 2]
```diff
@@ -1,69 +1,82 @@
...
-  });
-};
-
-addBook();
+    return book;
+  }
+}
 
-const getDataStored = JSON.parse(localStorage.getItem('data'));
+const removeBook = (deleteBtns) => {
```
Comment (bobb-Rob) [inline script.js]: Since I could not add book, I could not test remove book. Please ensure it works as expected

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Kaghenimbale, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Kaghenimbale) [reply script.js]: Hi Bobb,

I have try to clone again my project but I see that every thing is working perfectly.

Cheer

Kagheni Mbale Christian

Comment (bobb-Rob) [reply script.js]: The same issue persists even now.

**Comment by medaminedev66** [comment]
Hello @Kaghenimbale 👋,

_This is an additional review as a result of the second opinion request._

Regarding the issues raised, the code reviewer was correct. The user is still not able to add books. Before testing your app, please make sure you **checkout** to the same branch you submitted. Kindly ensure that your app works as expected before requesting a new review 😃.

Keep rocking 🚀!

**Review by algerina — APPROVED** [review]
Hi @Kaghenimbale  ,

Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**